packages/lib/src/spatial-navigation/components/virtualizedList/SpatialNavigationVirtualizedListWithVirtualNodes.tsx

```diff
@@ -162,13 +166,25 @@ export type SpatialNavigationVirtualizedListWithVirtualNodesProps<T> = Omit<
  * Framed letters correspond to rendered components.
  */
 export const SpatialNavigationVirtualizedListWithVirtualNodes = typedMemo(
-  <T extends ItemWithIndex>(props: SpatialNavigationVirtualizedListWithVirtualNodesProps<T>) => {
+  <T extends ItemWithIndex>(
+    props: SpatialNavigationVirtualizedListWithVirtualNodesProps<T> & {
+      idRef: React.Ref<SpatialNavigationVirtualizedListWithVirtualNodesRef>;
```

Comment:
Not a big fan of this new name either but it would be a little clearer

```suggestion
      getNodeIdRef: React.Ref<SpatialNavigationVirtualizedListWithVirtualNodesRef>;
```

Reply:
Fixed